ppr-ui/src/composables/mhrInformation/useTransferOwners.ts

```diff
@@ -404,7 +405,11 @@ export const useTransferOwners = (enableAllActions: boolean = false) => {
       }
       return showSuffixError
     }
+  }
 
+  // Transfer Affidavit flow and all the related conditions/logic
+  const TransAffidavit = {
+    affidavitCompleted: false
```

Comment:
Although a valid approach, it isn't totally consistent with what we've done here and after the recent pinia demo, sounds like the approach to preserve the state in store if possible.

Reply:
Hmm, yea I agree, let me rework it.